start-bsides-webserver.sh

```
@@ -0,0 +1,15 @@
+# Prepare
+docker network create --driver bridge businessnet 
+
+# Setup
+make docker
+docker run --name reverse-proxy --rm -p 80:80 -d -v ./nginx/reverse-proxy.conf:/etc/nginx/nginx.conf:ro --network businessnet nginx
+
+echo "Press enter to stop"
+read na
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)